docs/lob_guide/src/asciidoc/_chapters/working_with_lob.adoc

```diff
@@ -751,4 +751,380 @@ CQD `LOB_MAX_SIZE` (default 10G expressed in M [10000M]).
+  SALT USING 8 PARTITIONS
+;
+
+LOB Descriptor Chunks
+=====================
+
+CREATE TABLE TRAFODION.SCH."LOBDescChunks__04474425229029907479_0002"
+  (
+    DESCPARTNKEY                     LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , DESCSYSKEY                       LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKNUM                         INT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKLEN                         LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , DATAOFFSET                       LARGEINT DEFAULT NULL SERIALIZED
+  , STRINGPARAM                      VARCHAR(400) CHARACTER SET ISO88591 COLLATE DEFAULT DEFAULT NULL SERIALIZED
+  , PRIMARY KEY (DESCPARTNKEY ASC, DESCSYSKEY ASC, CHUNKNUM ASC)
+  )
+  SALT USING 8 PARTITIONS
+;
+
+************************************************
+LobNum: 3
+
+Data Storage
+============
+
+<External HDFS location>
+<External HDFS file>
+
+LOB Descriptor Handle
+=====================
+
+CREATE TABLE TRAFODION.SCH."LOBDescHandle__04474425229029907479_0003"
+  (
+    DESCPARTNKEY                     LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , NUMCHUNKS                        INT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , LOBLEN                           LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  )
+  STORE BY (DESCPARTNKEY ASC)
+  SALT USING 8 PARTITIONS
+;
+
+LOB Descriptor Chunks
+=====================
+
+CREATE TABLE TRAFODION.SCH."LOBDescChunks__04474425229029907479_0003"
+  (
+    DESCPARTNKEY                     LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , DESCSYSKEY                       LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKNUM                         INT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKLEN                         LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , DATAOFFSET                       LARGEINT DEFAULT NULL SERIALIZED
+  , STRINGPARAM                      VARCHAR(400) CHARACTER SET ISO88591 COLLATE DEFAULT DEFAULT NULL SERIALIZED
+  , PRIMARY KEY (DESCPARTNKEY ASC, DESCSYSKEY ASC, CHUNKNUM ASC)
+  )
+  SALT USING 8 PARTITIONS
+;
+
+--- SQL operation complete.
+```
+
+[#get statement for lob tables]
+== Get Statement for LOB Tables
+
+The GET Statement for lob tables gives column specific information on any lob columns in a table. 
+
+[#syntax]
+=== Syntax
+
+```
+GET LOB STATS FOR TABLE table-name;
+SELECT * FROM TABLE(LOB STATS table-name);
+```
+
+[#examples]
+=== Examples
+
+Suppose the following tables have been created:
+
+```
+>>CREATE TABLE tlob130gt (c1 INT NOT NULL, c2 BLOB, c3 CLOB, c4 BLOB, PRIMARY KEY (c1));
+
+--- SQL operation complete.
+
+>>CREATE TABLE tlob130gt2 (c1 INT NOT NULL, c2 BLOB, c3 CLOB, c4 BLOB STORAGE 'EXTERNAL', PRIMARY KEY (c1));
+
+--- SQL operation complete.
+```
+
+* This Get Statement displays statistics for the table tlob130gt, the lob information is formatted for machine readability.

```

Comment:
I'm confused. The example shows a SELECT statement, not a GET statement.

Reply:
Thanks Dave.
I've rewritten this part. :)